roles/openshift_common/tasks/main.yml

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
This file should be removed from the PR.

Reply:
@mtnbikenc can you please point where "os_use_contrail" variable is used if I remove this file?

Reply:
@savithruml why do we need this file?